### PR TITLE
feat: change input font size

### LINKED ISSUE #348: Changing fontSize of TextInput
TextInput Module fontSize doesn't get affected
### Current behaviour

fontSize remains 12 unaffected.

### Expected behaviour

fontSize should change right?? or am i missing something?

### Code sample

  <TextInput
        label='Email'
        value={this.state.text}
        onChangeText={text => this.setState({ text })}
       style={{flex:1, fontSize:16}}
      />

### Your Environment
Environment:
  OS: macOS Sierra 10.12.6
  Node: 8.8.1
  Yarn: 1.2.1
  npm: 5.4.2
  Watchman: 4.9.0
  Xcode: Xcode 9.1 Build version 9B46
  Android Studio: 3.1 AI-173.4697961

Packages: (wanted => installed)
  react: ^16.3.0-alpha.1 => 16.3.0-alpha.2
  react-native: ^0.54.2 => 0.54.2

### LINKED ISSUE #774: Placeholder and value are not in the same baseline on TextInput
### Current behaviour

In the efforts of aligning an icon with the text (like [here](https://storage.googleapis.com/spec-host-backup/mio-design%2Fassets%2F1Kezf2nmTVbQdOfNajrotDxeIKs1WlwHl%2Ftextfields-type-stroke.png)), I realized about that.

Currently is not possible to align an icon with the text.

### Expected behaviour

Placeholder line and value should be the same. So we can do things like aligning an icon:

mio-design_assets_1kezf2nmtvbqdofnajrotdxeiks1wlwhl_textfields-type-stroke

### Code sample

You can try that in our TextInput example.

### Screenshots (if applicable)

On the top with value and on the bottom with placeholder/label.

textinputalign

### Your Environment

| software              | version
| --------------------- | -------
| android   |  P
| react-native          | 0.57.8
| react-native-paper    | 2.4.0

### LINKED ISSUE #870: TextInput with multiline and without label has incorrect placeholder alignment on iOS
### Current behaviour

`TextInput` placeholder has incorrect alignment on iOS when used with `multiline={true}` and without `label`.

### Expected behaviour

Expect `TextInput` with `multiline={true}` and `placeholder` to look the same, whether with or without `label`.

Issue happens only on iOS, not on Android (see screenshots below).

### Code sample

```jsx
<TextInput
  placeholder={"Placeholder"}
  multiline={true}
  numberOfLines={1}
/>
```

### Screenshots (if applicable)

iOS  | Android 🤖
------------ | -------------
ios | android

Note: none of the inputs are focused, problem is on the first screenshot, second input.

### What have you tried

Have tried to set `textAlignVertical` to `center` but that's an Android-only property. Also tried to have the label as an empty string, which fix the alignment but hides the placeholder – which I need.

### Your Environment

| software              | version
| --------------------- | -------
| ios or android        | iOS
| react-native          | 0.57.8
| react-native-paper    | 2.10.1
| node                  | 11.9.0
| npm or yarn           | yarn
| expo sdk              | no

### LINKED ISSUE #877: Textinput height
### Current behaviour

If i change (make it smaller) the height of the TextInput component the placeholder text is not centered.

### Expected behaviour
A smaller textinput with the placeholder centered

### Code sample

<TextInput style={{width: '100%', height: 44}} mode='outlined' label='Test' />

### Your Environment

| software              | version
| --------------------- | -------
| ios or android        | android 7
| react-native          | 0.58.5
| react-native-paper    | 2.11
| node                  |
| npm or yarn           |
| expo sdk              |

### PR DESCRIPTION
**Pull request by jbinda**
Take fontSize from style object passed to TextInput to modify font size

fix #348 
fix #774 
fix #870
fix #877

### CODE REVIEW

**Comment by NomanGul** [comment]
Plz, merge this... I want font Size functionality/props

**Comment by Trancever** [comment]
Hey @jbinda What's the status of this PR, is it ready for review?

**Comment by jbinda** [comment]
@Trancever - it's almost done but I still deal with multiline. I will let you know soon.

File: example/src/Examples/TextInputExample.js [hunk 1]
```diff
@@ -77,13 +188,16 @@ class TextInputExample extends React.Component<Props, State> {
           />
           <View style={styles.inputContainerStyle}>
             <TextInput
-              label="Input with helper text"
+              label="Input with help"
```
Comment (Trancever) [inline example/src/Examples/TextInputExample.js]:
```suggestion
              label="Input with helper text"
```

File: src/components/TextInput/TextInput.js [hunk 2]
```diff
@@ -56,6 +56,10 @@ export type TextInputProps = {|
    * Whether to apply padding to label and input.
    */
   padding?: 'none' | 'normal',
+  /**
+   * Sets min height with densed layout.
```
Comment (Trancever) [inline src/components/TextInput/TextInput.js]: We need better description, especially how it behaves when height is also specified

Comment (Trancever) [reply src/components/TextInput/TextInput.js]: Also, let's add style prop description that mention that you can pass height there

**Comment by gawrysiak** [comment]
good job @jbinda on getting this merged 🎉

**Pull request merged**